databases/week2/More_queries_lesson2.sql

```diff
@@ -0,0 +1,34 @@
+-- 1.Get all the tasks assigned to users whose email ends in @spotify.com
+
+/*Select *  from task 
+inner join  user_task Where user_id in 
+(Select id From user where email like '%@spotify.com')*/
+
+SELECT *
+  FROM task
+  Inner JOIN user_task
+  ON task.id = user_task.task_id
+   Inner JOIN user
+  ON user_task.user_id = user.id
+  Where user.email like '%@spotify.com'
+
+  -- 2.Get all the tasks for 'Donald Duck' with status 'Not started'
+ select * 
```

Comment:
a suggestion , it will look better if you are consistent with the usage of upper case on the keywords